spring-cloud-context-integration-tests/pom.xml

```diff
@@ -61,8 +61,8 @@
 			<optional>true</optional>
 		</dependency>
 		<dependency>
-			<groupId>org.junit.vintage</groupId>
-			<artifactId>junit-vintage-engine</artifactId>
+			<groupId>org.junit.jupiter</groupId>
```

Comment:
This should be removed instead of changed. I believe it is part of spring-boot-starter-test.